Game/Source/GDKTestGymsFunctionalTests/Tests/CrossServerAbilityActivationTest/CrossServerAbilityActivationTest.cpp

```diff
@@ -24,6 +24,7 @@ ACrossServerAbilityActivationTest::ACrossServerAbilityActivationTest()
 {
 	Author = TEXT("Tilman Schmidt");
 	Description = TEXT("Tests that gameplay abilities can be activated across server boundaries.");
+	SetNumRequiredClients(1);
```

Comment:
This will be going away with Victoria's PR, but either way is fine, I think

Reply:
Might be that we want to clean out this API and calls to it as a follow-up to that PR then.